clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -79,14 +79,15 @@
                                            + " durability of records that are sent. The following settings are allowed: "
                                            + " <ul>"
                                            + " <li><code>acks=0</code> If set to zero then the producer will not wait for any acknowledgment from the"
-                                           + " server at all. The record will be immediately added to the socket buffer and considered sent. No guarantee can be"
+                                           + " server at all. The record will be immediately added to the socket buffer and considered sent. There is no guarantee"
+                                           + " that the socket buffer will be flushed prior to the shutdown of the producer JVM.  No guarantee can be"
```

Comment:
@ijuma yes, you're correct, my mistake.  I've rolled back the doc change and if KAFKA-3703 turns out to be fruitful in preventing data loss with acks=0 the docs should remain consistent this way as well.